src/core/qgsvectorlayerutils.h

```diff
@@ -162,11 +162,12 @@ class CORE_EXPORT QgsVectorLayerUtils
      * This should be used in scenarios, where a ``QWeakPointer<QgsVectorLayer>`` is kept in a thread
      * and features should be fetched from this layer. Using the layer directly is not safe to do.
      * The result will be ``nullptr`` if the layer has been deleted.
+     * If \a feedback is specified, the call will return if the feedback is canceled.
```

Comment:
`will return nullptr if the feedback is canceled`

Reply:
I was about to write this. But realized that this will also be a nullptr if the vectorLayer no longer exists, so this effect is not isolated to ...   I'll just add it in a different way